src/Form/Type/UiElement/ButtonLinkType.php

```diff
@@ -67,6 +68,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label' => 'monsieurbiz_richeditor_plugin.ui_element.monsieurbiz.button.field.link',
                 'constraints' => $constraintsLink,
             ])
+            ->add('target', ChoiceType::class, [
+                'choices'  => [
+                    'Open in the same frame as it was clicked (this is default)' => '_self',
+                    'Open in a new window or tab' => '_blank',
```

Comment:
Can you translate the labels ?

Reply:
Also can you use constants for `_self` and `_blank` ?

Reply:
doh! for sure. now that you point it out, sounds silly of me not to do this by default. thanks.